src/vector/index.html

```diff
@@ -34,18 +35,10 @@
         <% }
     } %>
   </head>
-  <body style="height: 100%;">
+  <body style="height: 100%;" data-vector-indexeddb-worker-script = '<%= htmlWebpackPlugin.files.chunks["indexeddb-worker"].entry %>'>
```

Comment:
Slight tweak suggested to match attribute style:

```suggestion
  <body style="height: 100%;" data-vector-indexeddb-worker-script="<%= htmlWebpackPlugin.files.chunks['indexeddb-worker'].entry %>">
```